CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to Open Source Studio at ITP
+
+## Who can contribute?
+**Everyone** is welcome to engage in the open source project that is the organisation you are currently viewing. We encourage discussion and contribution from outside the student body of the class. 
+
+The only limitation as an outside collaborator is the actual class meeting space, a seminar-style meeting in person. This is a private space for the enrolled students to discuss, question, and collaborate on assignments around the practice and philosophy of open source.
+
```

Comment:
does this make us the _elite_ group of this project? maybe we should have a section or repo or document that summarizes the main talking points from class so that anyone contributing has access to our discussions, questions, line of thinking etc

Reply:
> anyone contributing has access to our discussions, questions, line of thinking etc

As an outside collaborator and a beginner in open source practices, I would extremely appreciate this!!!

Reply:
@doodybrains this is a great idea. I think the blog posts are a nice start to this, but if someone wants to volunteer to take notes and post recaps of class (keeping in mind student privacy) this would be welcome!

Reply:
i would be down to do this for a class or two - i'll admit i'm a terrible note taker but its something i'm interested in getting better at. maybe we could set up some sort of living document outside github?

Reply:
I was about to suggest Hackpad but looks like it was closed down!

https://github.com/hackpad/hackpad

But yes, something like Google docs or similar real-time collaborative document could work. 

There's a side conversation also to be had about how typing can appear as "rude" / "not paying attention" while someone is speaking but handwritten note-taking can be a signal of "paying attention." I typed notes during @tchoi8's presentation b/c I had forgotten my notepad and felt uncomfortable doing so.